src/components/organisms/footer/footer.html

```diff
@@ -0,0 +1,37 @@
+<div class="footer">
+  <div class="contact">
+    <div class="footer-img">
+      <img src="../../../img/footer.png" />
+    </div>
```

Comment:
```suggestion
    <div class="footer-img-wrapper">
      <img src="../../../img/footer.png" />
    </div>
```
`img` とクラス名に入っているとその要素が `img` 要素そのものだと誤解される可能性があるので、このように `img` を囲んでいる要素であることを明示すると可読性が向上します。

また、そもそもラッパー要素を省くことはできないかどうか考えるのも良いと思います。